source/stylesheets/_rebo.scss

```diff
@@ -194,3 +194,22 @@ a.rebo {
 		@include size(8px);
 	}
 }
+
+.rebo.inline {
```

Comment:
Move it to another file and call it another class, this class is reserved for that one specific button style we use in most of the cases.